recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
-    '''
-    complete the code to compute a variable movie_ratings
```

Comment:
It is recommended to use more descriptive variable names. Consider using 'movies_ratings_merged' instead of 'merged_data'.